src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/index.tsx

```diff
@@ -142,34 +147,31 @@ class Form extends React.Component<Props<Rule, KeysOfUnion<Rule>>, State> {
                 name="pattern"
                 placeholder={t('[a-zA-Z0-9]+')}
                 onChange={this.handleChange('pattern')}
-                value={rule.pattern}
+                value={values.pattern}
                 onBlur={onValidate('pattern')}
               />
             </Field>
           )}
         </FieldGroup>
-        {onUpdateEventId && (
-          <React.Fragment>
-            <ToggleWrapper>
-              {displayEventId ? (
-                <Toggle priority="link" onClick={this.handleToggleEventId}>
-                  {t('Hide Event ID field')}
-                  <IconChevron direction="up" size="xs" />
-                </Toggle>
-              ) : (
-                <Toggle priority="link" onClick={this.handleToggleEventId}>
-                  {t('Use Event ID for auto-completion')}
-                  <IconChevron direction="down" size="xs" />
-                </Toggle>
-              )}
-            </ToggleWrapper>
-          </React.Fragment>
-        )}
+        <ToggleWrapper>
+          {displayEventId ? (
+            <Toggle priority="link" onClick={this.handleToggleEventId}>
+              {t('Hide Event ID field')}
```

Comment:
Could we use `Event ID` or `event ID`? You have a mix of both right now.

Reply:
I left the capital letter 'e' from Event only when it is the first word of a sentence 😉